tests/parquet_test.py

```diff
@@ -515,13 +515,16 @@ def test_multicol_write(self):
             "c_6": ak.SegArray(ak.array([0, 5, 10]), ak.randint(0, 1, 15, dtype=ak.bool)),
             "c_7": ak.array(np.random.uniform(0, 100, 3)),
             "c_8": ak.SegArray(ak.array([0, 9, 14]), ak.array(np.random.uniform(0, 100, 20))),
-            "c_9": ak.array(["abc", "123", "xyz"])
+            "c_9": ak.array(["abc", "123", "xyz"]),
+            "c_10": ak.SegArray(ak.array([0, 2, 5]), ak.array(["a", "b", "c", "d", "e", "f", "g", "h", "i"])),
+            "c_11": ak.SegArray(ak.array([0, 2, 2]), ak.array(["a", "b", "c", "d", "e", "f", "g", "h", "i"])),
+            "c_12": ak.SegArray(ak.array([0, 0, 2]), ak.array(["a", "b", "c", "d", "e", "f", "g", "h", "i"])),
+            "c_13": ak.SegArray(ak.array([0, 5, 8]), ak.array(["a", "b", "c", "d", "e", "f", "g", "h", "i"])),
```

Comment:
should we maybe update one of these to have strings that are longer than 1 character to be sure that functions as expected?

Reply:
Probably would not hurt. I added an extra column to test this.